roles/translator/src/lib/downstream_sv1/diff_management.rs

```diff
@@ -342,13 +342,13 @@ mod test {
         }
 
         let calculated_share_per_min = count as f32 / (elapsed.as_secs_f32() / 60.0);
-        // This is the error margin for a confidence of 99% given the expect number of shares per
+        // This is the error margin for a confidence of 99.99% given the expect number of shares per
```

Comment:
```suggestion
        // This is the error margin for a confidence of 99.99...% given the expect number of shares per
```

99.99% still gives some impression of exactness, which is not the case

we should use a (`...`) to imply recurring decimals and make it clear we are really overshooting